Add section for Builder Versioning (#871)
zeit/docs: pages/docs/v2/deployments/builders/overview.mdx

<Note>It is important to remember that the <InlineCode>src</InlineCode> property is a glob. In the above example, this will only use <InlineCode>@now/static</InlineCode> for the HTML files in the root directory. If you need it to be used throughout subdirectories, you need the appropriate glob syntax: <InlineCode>**/*.html</InlineCode>.</Note>

## Builder Versioning

> Builders are published to [npmjs.com](https://npmjs.com) and installed at build time. The `use` statement in your `now.json` file uses a similar syntax to `npm install`. This means versioning works the same for Builders as it does for any npm package.

Author: ```suggestion
All of the Official Builders are published to [npmjs.com](https://npmjs.com) and installed at build time. The `use` statement in your `now.json` file uses a similar syntax to `npm install`. This means versioning works the same for Builders as it does for any npm package.
```

> All of the Official Builders are published to [npmjs.com](https://npmjs.com) and installed at build time. The `use` statement in your `now.json` file uses a similar syntax to `npm install`. This means versioning works the same for Builders as it does for any npm package.

Reviewer: ```suggestion
Official Builders are published to [npmjs.com](https://npmjs.com) and installed [at build time](/docs/v2/deployments/concepts/lifecycle/#builds-lifecycle). The `use` statement in your [`now.json` file](/docs/v2/deployments/configuration) uses a similar syntax to `npm install` or `yarn add`. This means versioning works the same for Builders as it does for any npm module, by utilising a scope, e.g., `@now/node@0.7.3`.
```

> Official Builders are published to [npmjs.com](https://npmjs.com) and installed [at build time](/docs/v2/deployments/concepts/lifecycle/#builds-lifecycle). The `use` statement in your [`now.json` file](/docs/v2/deployments/configuration) uses a similar syntax to `npm install` or `yarn add`. This means versioning works the same for Builders as it does for any npm module, by utilising a scope, e.g., `@now/node@0.7.3`.

Author: ```suggestion
Official Builders are published to [npmjs.com](https://npmjs.com) and installed [at build time](/docs/v2/deployments/concepts/lifecycle/#builds-lifecycle). The `use` statement in your [`now.json` file](/docs/v2/deployments/configuration) uses a similar syntax to `npm install` or `yarn add`. This means versioning works the same for Builders as it does for any npm package.
```

> Official Builders are published to [npmjs.com](https://npmjs.com) and installed [at build time](/docs/v2/deployments/concepts/lifecycle/#builds-lifecycle). The `use` statement in your [`now.json` file](/docs/v2/deployments/configuration) uses a similar syntax to `npm install` or `yarn add`. This means versioning works the same for Builders as it does for any npm module, by utilising a scope, e.g., `@now/node@0.7.3`.

Author: ```suggestion
Official Builders are published to [npmjs.com](https://npmjs.com) and installed [at build time](/docs/v2/deployments/concepts/lifecycle/#builds-lifecycle). The `use` statement in your [`now.json` file](/docs/v2/deployments/configuration) uses a similar syntax to `npm install` or `yarn add`. This means versioning works the same for Builders as it does for any npm package.
```

@timothyis I think you don't need to provide an example here because we have them below. Also we don't really need to introduce the user to `scope` since its not necessary to understand versioning, so I removed it.

> The follow are valid examples you might use to pin a verison of a [@now/node](https://www.npmjs.com/package/@now/node?activeTab=versions) in your `now.json` configuration file.

Reviewer: ```suggestion
The following are valid examples you can use to pin a version of [@now/node](https://www.npmjs.com/package/@now/node?activeTab=versions) in your [`now.json` configuration file](/docs/v2/deployments/configuration/#builds).
```

- `@now/node`
- `@now/node@latest`
- `@now/node@0.7.3
- `@now/node@canary`
- `@now/node@0.7.2-canary.2`

> All of the Official Builders have a Stable branch and a Canary branch that are published to npm as `latest` and `canary` [dist-tags](https://docs.npmjs.com/adding-dist-tags-to-packages) respectively. Features and fixes will be published to Canary before they reach Stable. If you don't specify a specific version or dist-tag, then `latest` is assumed.

Reviewer: ```suggestion
All Official Builders have a Stable branch and a Canary branch that are published to npm as `latest` and `canary` ["dist-tags"](https://docs.npmjs.com/adding-dist-tags-to-packages) respectively. Features and fixes will be published to Canary before they reach Stable. If you do not specify a specific version or dist-tag, then the `latest` version is assumed.
```
